app/assets/javascripts/components/toggle_switch.jsx

```diff
@@ -1,23 +0,0 @@
-import { Component } from 'react'
```

Comment:
It seems useful to extract this component rather than put it in a render() method.

Reply:
This "toggle" component has not been moved into a render() method. It has been completely removed. The UX for it is rather clunky and until it can be replaced with something more elegant it is better to have two simple buttons that will function in it's place.

Reply:
+1